app/views/welcome/_user_feed.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="row">
```

Comment:
Maybe call it just `_feed.html.erb` instead if we don't care about a user role/type?